andela/lotus-ah-frontend
# #161676743 User Logout Feature

#### What does this PR do?
 - Implemented the logout feature

#### Description of Task to be completed?
- a user should be able to log out of the application

#### How should this be manually tested?
- start the app by running ```yarn start:dev```
the app should open on port ```8080``` on your browser
navigate to the login page, login as a user then use the logout 
link to log out of the app

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#161676743](https://www.pivotaltracker.com/story/show/161676743, "Users should be able to Logout of Authors Haven")

#### Screenshots (if appropriate)


#### Questions:
